strapi/gcp-deployment/package.json

```diff
@@ -8,7 +8,10 @@
     "start": "strapi start",
     "build": "strapi build",
     "strapi": "strapi",
-    "gcp-build": "strapi build"
+    "gcp-build": "strapi build",
+    "app-deploy": "gcloud app deploy app.yaml --project=ch-test-strapi --bucket=gs://ch-test-strapi-app-bucket",
+    "terraform-deploy-init": "terraform apply -var-file=\"secret.tfvars\" -target=google_storage_bucket.terraform_state",
+    "terraform-deploy": "terraform apply -var-file=\"secret.tfvars\""
```

Comment:
This directory structure of `strapi` + `terraform` in `gcp-deployment` works well for this experiment. For production, it's more typical to have the Terraform config for the whole environment (staging/production) in its own directory, and to have apps in a separate environment. See also https://github.com/bettersg/call-home/pull/69